packages/pangea-node-sdk/package.json

```diff
@@ -18,29 +21,30 @@
     "merkle-tools": "^1.4.1"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.22.5",
-    "@gitlab/eslint-plugin": "^19.0.0",
+    "@gitlab/eslint-plugin": "^19.3.0",
     "@types/crypto-js": "^4.2.0",
     "@types/jest": "29.5.3",
-    "@typescript-eslint/eslint-plugin": "^5.35.1",
-    "@typescript-eslint/parser": "^5.35.1",
+    "@typescript-eslint/eslint-plugin": "^6.15.0",
+    "@typescript-eslint/parser": "^6.15.0",
     "coverage": "^0.4.1",
-    "eslint": "^8.23.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-prettier": "^4.0.0",
+    "cross-env": "^7.0.3",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^5.1.0",
     "jest": "29.6.2",
-    "prettier": "^2.6.0",
+    "prettier": "^3.1.1",
     "ts-jest": "^29.1.0",
     "ts-node": "^9.1.1",
     "tsc-alias": "^1.8.6",
     "typedoc": "^0.24.8",
     "typescript": "^5.1.6"
   },
   "scripts": {
-    "test": "NODE_OPTIONS=--experimental-vm-modules npx jest",
-    "test:unit": "NODE_OPTIONS='--experimental-vm-modules --openssl-legacy-provider' npx jest --testPathPattern=tests/unit",
-    "test:integration": "NODE_OPTIONS='--experimental-vm-modules --openssl-legacy-provider' npx jest --testPathPattern=tests/integration",
-    "lint": "npx prettier --check . && npx eslint --ext .ts",
+    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules npx jest",
+    "test:unit": "cross-env NODE_OPTIONS=\"--experimental-vm-modules --openssl-legacy-provider\" npx jest --testPathPattern=tests/unit",
+    "test:integration": "cross-env NODE_OPTIONS=\"--experimental-vm-modules --openssl-legacy-provider\" npx jest --testPathPattern=tests/integration",
+    "lint": "npx prettier --check . && npx eslint --ext .ts .",
```

Comment:
ESLint was **not** running at all because this was missing an argument for _what_ to lint.